templates/settings.yml

```
@@ -0,0 +1,104 @@
+# These settings are synced to GitHub by https://probot.github.io/apps/settings/
+repository:
+  homepage: "https://developer.lightbend.com/start/?group=play"
+  topics: playframework, example, example-project, sample, sample-app, jvm, webapp
+  private: false
+  has_issues: true
+  # We don't need projects in sample projects
+  has_projects: false
+  # We don't need wiki in sample projects
+  has_wiki: false
+  has_downloads: true
+  default_branch: 2.7.x
+  allow_squash_merge: true
+  allow_merge_commit: false
+  allow_rebase_merge: false
+
+teams:
+  - name: core
+    permission: admin
+  - name: integrators
+    permission: write
+  - name: write-bots
+    permission: write
+
+branches:
+  - name: "[0-9].*.x"
+    protection:
+      # We don't require reviews for sample applications because they are mainly
+      # updated by template-control, which is an automated process
+      required_pull_request_reviews: null
+      # Required. Require status checks to pass before merging. Set to null to disable
+      required_status_checks:
+        # Required. The list of status checks to require in order to merge into this branch
+        contexts: ["Travis CI - Pull Request", "WIP", "typesafe-cla-validator"]
```

```suggestion
        contexts: ["Travis CI - Pull Request", "typesafe-cla-validator"]
```

I removed WIP bot because Mergify doesn't require it.  We need reproduce it as a Mergify rule.